content/en/users/aai/check-in/linking/_index.md

```diff
@@ -65,58 +79,26 @@ browser.
 
 To link a subject DN to your EGI account:
 
-1. Enter the following URL in a browser: <https://aai.egi.eu/registry>
+1. Make sure your certificate is already imported in your browser. 
 
-1. Click **Login** and authenticate using any of the login credentials _already_
-   linked to your EGI account
-
-1. Navigate to **My EGI User Community Identity** page in one of the following
-   ways:
-
-   - hover over your display name next to the gear icon on the top right corner
-     of the page; _or, alternatively,_
-   - select **EGI User Community** from the list of available Collaborations and
-     then click **My EGI User Community Identity** from the **People** menu
-
-   ![Check-in my identity](./check-in-my-identity.png)
-
-1. Under the **Organisational Identities** section of your profile page, expand
-   **Actions** menu and click **Link New Identity**.
+2. Follow the same steps as above (login, go to profile, click ‘Link New Identity’)
```

Comment:
go to "Account Security" --> "Linked identities" and select "Link account"

<img width="1175" alt="Screenshot 2025-05-27 at 14 41 14" src="https://github.com/user-attachments/assets/f0180985-be00-42cc-908a-1ef976614c4c" />